packages/node-http-handler/src/set-socket-keep-alive.ts

```diff
@@ -0,0 +1,16 @@
+import { ClientRequest } from "http";
+
+export interface SocketKeepAliveOptions {
+  keepAlive: boolean;
+  keepAliveMsecs?: number;
+}
+
+export const setSocketKeepAlive = (request: ClientRequest, socketKeepAlive: SocketKeepAliveOptions) => {
+  if (socketKeepAlive.keepAlive !== true) {
+    return;
+  }
+
+  request.on("socket", (socket) => {
+    socket.setKeepAlive(socketKeepAlive.keepAlive, socketKeepAlive.keepAliveMsecs || 0);
+  });
+};
```

Comment:
Use object destructuring to remove variable `socketKeepAlive`
```suggestion
export const setSocketKeepAlive = (request: ClientRequest, { keepAlive, keepAliveMsecs }: SocketKeepAliveOptions) => {
  if (keepAlive !== true) {
    return;
  }

  request.on("socket", (socket) => {
    socket.setKeepAlive(keepAlive, keepAliveMsecs || 0);
  });
};
```